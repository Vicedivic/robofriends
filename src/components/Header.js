import React, { memo } from "react";

const Header = () => {
  return <h1 className="f1">RoboFriends - Where Robots Meet</h1>;
};

export default memo(Header);
