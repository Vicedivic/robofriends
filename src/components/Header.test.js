import React from "react";
import ShallowRenderer from "react-shallow-renderer";
import Header from "./Header";

describe("Header", () => {
  const renderer = new ShallowRenderer();
  const headerHtml = renderer.render(<Header />);

  it("renders properly", () => {
    expect(headerHtml).toMatchSnapshot();
  });
});