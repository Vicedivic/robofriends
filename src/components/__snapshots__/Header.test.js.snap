// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders properly 1`] = `
<h1
  className="f1"
>
  RoboFriends - Where Robots Meet
</h1>
`;
